index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/styles.css">
+    <title> The Good Place Application</title>
+</head>
+<body>
+    <header>
+        <h1>Getting into The Good Place...</h1>
+    </header>
+    <img src="assets/sky.jpg" alt="image of the sky">
+    <h2>So you want to get into The Good Place?</h2>
+    <p>Tough luck, you'll need to fill out a form for that. 
+        And then another, and probably one hundred more after that. 
+        This is the first one.</p>
```

Comment:
It's a good idea to just return this closing p tag since you have a paragraph here.